packages/mux-player/src/themes/gerwig/gerwig.html

```diff
@@ -90,9 +90,10 @@
         var(--_accent-color) 25%,
         var(--_accent-color)
       );
-      --media-range-thumb-width: 0;
-      --media-range-thumb-height: 0;
-      --media-range-thumb-transition: all 0.3s;
+      --media-range-thumb-width: 12px;
+      --media-range-thumb-height: 12px;
+      --media-range-thumb-transform: scale(0);
+      --media-range-thumb-transition: transform 0.3s;
```

Comment:
transform scale is preferred over width, height because it's centered by default and performs better.